android/src/main/java/co/nimblehq/ic/kmm/suv/android/di/ViewModelModule.kt

```diff
@@ -0,0 +1,9 @@
+package co.nimblehq.ic.kmm.suv.android.di
+
+import co.nimblehq.ic.kmm.suv.android.ui.screens.login.LoginViewModel
+import org.koin.androidx.viewmodel.dsl.viewModel
+import org.koin.dsl.module
+
+val viewModelModule = module {
+    viewModel { LoginViewModel(get()) }
+}
```

Comment:
Do you intend to define all the factories for ViewModel instances in this file? If not, I'd suggest rename this file based on screen/feature scope, ex: `loginModude`, `homeModule`, `detailModule`, etc.

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/92/commits/c3e5eee44a768ed400b77ebb78d1ae4ab69c1567